front/src/containers/Authentication/styleLogin.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components'
+
+export const Container = styled.SafeAreaView`
+  flex: 1;
+  background: #333;
+  justify-content: center;
+  align-items: center;
+`
+export const ContainerView = styled.View `
+  flex: 1;
+`
+
+export const ContainerLogo = styled.View `
+  margin: 20% auto;
+`
+
+export const ContainerItem = styled.View`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`
+
+export const Logo = styled.Image`
+  width: 150px;
+  height: 150px;
+`
+
+export const InputField = styled.TextInput.attrs({
+  placeholderTextColor: '#fff'
+}) `
+  width: 250px;
+  border: none;
+  border-bottom-color: #fff;
+  border-bottom-width: 2px;
+  margin: 15px 0;
+  padding: 1px;
+  color: #fff;
+`
+
+export const HyperLink = styled.Text`
+  color: #fff;
+  margin: 20% auto;
+  text-decoration: underline;
+`
+
+export const Icon = styled.View `
```

Comment:
I think that this style can be applied in own icon component and not create a view